edn/tests/tests.rs

```diff
@@ -53,6 +53,7 @@ fn test_nil() {
 
 #[test]
 fn test_nan() {
+    assert_eq!(nan("#fNaN").unwrap(), Float(OrderedFloat(f64::NAN)));
```

Comment:
@rnewman, makes sense. If that's the case, whitespace should be enforced. Is that the case for + and -Infinity as well?